Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/Cancel.hs

```diff
@@ -17,21 +17,18 @@ module Beckn.ACL.Cancel (buildCancelReq, buildCancelSearchReq) where
 import qualified Beckn.Types.Core.Taxi.Cancel.Req as Cancel
 import qualified Domain.Action.UI.Cancel as DCancel
 import qualified Domain.Types.BookingCancellationReason as SBCR
-import Environment
 import Kernel.Prelude
 import qualified Kernel.Types.Beckn.Context as Context
 import Kernel.Types.Beckn.ReqTypes
 import Kernel.Utils.Common
 
 buildCancelReq ::
-  (HasFlowEnv m r ["bapSelfIds" ::: BAPs Text, "bapSelfURIs" ::: BAPs BaseUrl]) =>
+  (MonadFlow m, MonadReader r m) =>
```

Comment:
here and in other places `MonadReader r m `is redundant

Reply:
Done.